tests/python/relay/test_op_level3.py

```diff
@@ -903,8 +904,8 @@ def verify_scatter(dshape, ishape, axis=0):
         indices_np = np.random.randint(-dshape[axis], dshape[axis] - 1, ishape).astype("int64")
 
         ref_res = ref_scatter(data_np, indices_np, updates_np, axis)
-        # TODO(mbrookhart): expand testing when adding more backend schedules
-        for target, ctx in [("llvm", tvm.cpu())]:
+
```

Comment:
maybe we can add a test to dyn op_level3 as well since the shape func is elemwise?

Reply:
Hmm, that's a good idea. I'm not sure the dyn namespace is the right place though? Maybe just add a second test with dynamic shapes here? or in test_any?

Reply:
yeah, I think test_any is already pretty large. But we can probably put it there for now. We may need to split the tests later as well.

Reply:
@zhiics, added dynamic-shaped tests to test_op_level3.py. It was pretty painless, everything worked :+1: